building failed!

ubgpu@ubgpu:~/github/ctr$ make linux
g++ -Wall -I/home/chongw/include -L/home/chongw/lib main.cpp utils.cpp corpus.cpp ctr.cpp data.cpp opt.cpp -o ctr-condor -lgsl -lm -lgslcblas
utils.cpp: In function ‘void sym_eigen(gsl_matrix_, gsl_vector_, gsl_matrix_)’:
utils.cpp:185:7: warning: variable ‘r’ set but not used [-Wunused-but-set-variable]
   int r;
       ^
In file included from corpus.cpp:1:0:
corpus.h: In constructor ‘c_document::c_document()’:
corpus.h:20:15: error: ‘NULL’ was not declared in this scope
     m_words = NULL;
               ^
corpus.h: In destructor ‘c_document::~c_document()’:
corpus.h:34:20: error: ‘NULL’ was not declared in this scope
     if (m_words != NULL) {
                    ^
make: *_\* [linux] Error 1
ubgpu@ubgpu:~/github/ctr$

--- a/corpus.h
+++ b/corpus.h
@@ -4,6 +4,7 @@
 #define CORPUS_H
 
 #include <vector>
+#include <stddef.h>
 
 using namespace std;
 

